### ISSUE TITLE
Authentication Failure with GitHub Desktop after installing pre-commit

### ISSUE THREAD

**Issue by anselmbradford** [desktop/desktop#15671]
### The problem

I ran `pip install -U pre-commit && pre-commit install` and immediately afterward get this dialog box when trying to make a commit in GitHub Desktop:

<img width="600" alt="Screen Shot 2022-11-23 at 11 02 29 AM" src="https://user-images.githubusercontent.com/704760/203596124-cafa853c-e448-4d18-b18c-b43b48647f0f.png">

### Release version

3.1.2

### Operating system

Mac OSX 12.6

### Steps to reproduce the behavior

1. `pip install -U pre-commit && pre-commit install`
2. Try to make a commit in GitHub Desktop

**Comment by anselmbradford** [comment]
Note: I have attempted the steps [here](https://github.com/desktop/desktop/issues/8860#issuecomment-1312476492), but they didn't resolve it.

**Comment by anselmbradford** [comment]
Figured it out. Feel free to close this issue. For reference below is what the issue was. Perhaps this scenario could be included in the error dialog box? 

In `.pre-commit-config.yaml` I had the line `repo: https://gitlab.com/pycqa/flake8`. GitLab requires a login, so this was preventing a commit using GitHub Desktop. I discovered this issue when running pre-commit on its own with `pre-commit run --all-files` and it asked for a login for GitLab.
